module/VuFind/src/VuFind/Db/Entity/UserListEntityInterface.php

```diff
@@ -40,4 +42,90 @@
  */
 interface UserListEntityInterface extends EntityInterface
 {
+    /**
+     * Get identifier (returns null for an uninitialized or non-persisted object).
+     *
+     * @return ?int
+     */
+    public function getId(): ?int;
+
+    /**
+     * Title setter
+     *
+     * @param string $title Title
+     *
+     * @return UserListEntityInterface
+     */
+    public function setTitle(string $title): UserListEntityInterface;
+
+    /**
+     * Get Title.
+     *
+     * @return string
+     */
+    public function getTitle(): string;
+
+    /**
+     * Description setter
+     *
+     * @param string $description Description
+     *
+     * @return UserListEntityInterface
+     */
+    public function setDescription(string $description): UserListEntityInterface;
+
+    /**
+     * Get Description.
+     *
+     * @return string
+     */
+    public function getDescription(): string;
+
+    /**
+     * Created date setter.
+     *
+     * @param DateTime $dateTime Created date
+     *
+     * @return UserListEntityInterface
+     */
+    public function setCreated(DateTime $dateTime): UserListEntityInterface;
+
+    /**
+     * Get created date.
+     *
+     * @return DateTime
+     */
+    public function getCreated(): DateTime;
+
+    /**
+     * Set the list public.
+     *
+     * @param bool $public Created date
+     *
+     * @return UserListEntityInterface
+     */
+    public function setPublic(bool $public): UserListEntityInterface;
+
+    /**
+     * Check if the list is public
+     *
+     * @return bool
+     */
+    public function isPublic(): bool;
+
+    /**
+     * Set user ID.
```

Comment:
This should be "Set user," not "Set user ID," since it's taking in a whole object.